src/pages/InfoList/InfoListForm.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import InfoFeed from './InfoFeed';
+
+const InfoListForm = () => {
+  const [infoList, setInfoList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/infoListData.json')
+      .then(res => res.json())
+      .then(data => {
+        let { infoList } = data.message;
+        setInfoList([...infoList]);
+      });
+  }, [infoList]);
+
+  return (
+    <div>
+      {infoList &&
+        infoList.map(
```

Comment:
```suggestion
      {infoList.map(
```
이렇게 써도 문제 없는데 왜 그럴까요?

Reply:
앗 제가 불필요한 코드를 추가했군요! 수정하도록 하겠습니다! 감사합니다